domains/transfer/Transfer.API/API/Shared/Swagger/ConfigureSwaggerOptions.cs

```diff
@@ -0,0 +1,27 @@
+using Asp.Versioning.ApiExplorer;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace API.Shared.Swagger;
+
+public class ConfigureSwaggerOptions : IConfigureOptions<SwaggerGenOptions>
```

Comment:
Why is this not set in Program.cs with all the other optiosn for SwaggerGen?

Reply:
I can easily move it to program.cs as a internal class. Is that what you want?

Reply:
All swagger setup is in this file now